sound/soc/sof/intel/hda-stream.c

```diff
@@ -234,6 +236,17 @@ int hda_dsp_stream_trigger(struct snd_sof_dev *sdev,
 					SOF_HDA_SD_CTL_DMA_START |
 					SOF_HDA_CL_DMA_SD_INT_MASK);
 
+		ret = snd_sof_dsp_read_poll_timeout(sdev, HDA_DSP_HDA_BAR,
+						    sd_offset, run, ((run &
+						    SOF_HDA_SD_CTL_DMA_START)
+						    ==
+						    SOF_HDA_SD_CTL_DMA_START),
```

Comment:
@zhuyingjiang Having just "==" on the line is not readable anymore. I'd recommend not aligning to the opening brace and having a bit more space for the arguemnts. E.g. see call to snd_sof_dsp_read_poll_timeout() in hda-dsp.c:hda_dsp_core_power_down().

Reply:
@kv2019i OK, thanks!